proto3-suite.cabal

```diff
@@ -80,6 +80,7 @@ library
                        Proto3.Suite.DotProto.Generate.Record
                        Proto3.Suite.DotProto.Generate.Syntax
                        Proto3.Suite.DotProto.Parsing
+                       Proto3.Suite.DotProto.TH
```

Comment:
Do you anticipate this being used by clients of this package?  I thought it was just an aid in testing.

Reply:
My philsophy when designing libraries is to exposed everything because I've been stopped in my tracks too many times before by not being able to import something I wanted to import from a package. So right now I just see it being used for testing, but I may as well exposed it should anyone else want to make use of the feature. 

Should I document that it is only intended for testing, or hide it from users?

Reply:
If it is currently only for testing, then I think it should be in the testing executable, not in the library.

The down side of exposing too much is that it's a breaking change to change or remove a feature once it has been exposed for general use.  If it really is just for testing, then keeping it as part of the testing package makes it easier to change how testing is done in the future, because (presumably) changing the testing executable is not a breaking change.